src/Uno.UI/WasmCSS/Uno.UI.css

```diff
@@ -159,6 +159,19 @@ embed.uno-frameworkelement.uno-unarranged {
     cursor: text;
 }
 
+input:-webkit-autofill,
+input:-webkit-autofill:hover,
+input:-webkit-autofill:focus,
+textarea:-webkit-autofill,
+textarea:-webkit-autofill:hover,
+textarea:-webkit-autofill:focus,
```

Comment:
Not sure if it's better to combine `input` and `textarea` into `.uno-textboxview`.